config/database.yml

```diff
@@ -0,0 +1,31 @@
+# SQLite. Versions 3.8.0 and up are supported.
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem "sqlite3"
+#
+default: &default
+  adapter: sqlite3
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  timeout: 5000
+
+development:
+  <<: *default
+  database: db/development.sqlite3
+  username: postgres
+  password: db_password
+
+# Warning: The database defined as "test" will be erased and
+# re-generated from your development database when you run "rake".
+# Do not set this db to the same as development or production.
+test:
+  <<: *default
+  database: db/test.sqlite3
+  username: postgres
+  password: db_password
+
```

Comment:
Please note that as part of the requirement for this project, you should use Postgres as the database for the application. 👍🏽

Reply:
Kindly ensure your App has unit tests and integration tests for all the most important components of the RoR application 😉